modules/gapi/src/api/kernels_core.cpp

```diff
@@ -391,17 +391,17 @@ core::GDetections parseSSD(const GMat& in,
 }
 
 core::GRects parseSSD(const GMat& in,
-                      const GOpaque<Size> & in_sz,
+                      const GOpaque<Size>& in_sz,
                       const float& confidence_threshold,
                       const bool& filter_out_of_bounds) {
     return core::GParseSSD::on(in, in_sz, confidence_threshold, filter_out_of_bounds);
 }
 
 core::GDetections parseYolo(const GMat& in,
-                      const GOpaque<Size>& in_sz,
-                      float confidence_threshold,
-                      float nms_threshold,
-                      const core::GYoloAnchors& anchors) {
+                            const GOpaque<Size>& in_sz,
+                            const float& confidence_threshold,
+                            const float& nms_threshold,
```

Comment:
I don't think it's necessary to give `float` variables by reference.. up to you :)

Reply:
Necessary)